layersvt/device_simulation.cpp

```diff
@@ -986,7 +1245,7 @@ VKAPI_ATTR void VKAPI_CALL GetPhysicalDeviceMemoryProperties(VkPhysicalDevice ph
     const auto dt = instance_dispatch_table(physicalDevice);
 
     PhysicalDeviceData *pdd = PhysicalDeviceData::Find(physicalDevice);
-    DebugPrintf("GetPhysicalDeviceMemoryProperties physicalDevice %p pdd %p\n", physicalDevice, pdd);
+    // DebugPrintf("GetPhysicalDeviceMemoryProperties physicalDevice %p pdd %p\n", physicalDevice, pdd);
```

Comment:
Not a huge fan of commented out code, do we need some sort of log level to deal with debug verbosity?

Reply:
Yes, please kill it with fire.

Reply:
Please kill it with fire.

Reply:
I leave those in because they're really for my use only, and it was convenient to re-enable as needed.  There's already a boolean to enable debug messages (which was getting spammed), I thought log levels would be overkill.
Will remove.

Reply:
Gone, in a poof of smoke